src/screen/DeeplinkNavigatorScreen.tsx

```diff
@@ -0,0 +1,34 @@
+import {NavigationProp, RouteProp, useLinkTo} from '@react-navigation/native';
+import {ChainApiContext} from 'context/ChainApiContext';
+import {NetworkContext} from 'context/NetworkContext';
+import LoadingView from 'presentational/LoadingView';
+import React, {useContext} from 'react';
+import {AppStackParamList} from 'src/navigation/navigation';
+import {deeplinkNavigatorScreen} from 'src/navigation/routeKeys';
+
+// TODO: Remove this file and move the logic to push notification handling function
```

Comment:
I couldn't test this properly without a build. after the build I can fix this point in another PR.

Reply:
Let's create an issue so we don't forget

Reply:
#258